testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractBrokerTest.java

```diff
@@ -113,22 +115,37 @@ protected IdentityProviderRepresentation createIdentityProvider(String alias, St
         return identityProviderRepresentation;
     }
 
+    private void waitForPage(String title) {
+        long startAt = System.currentTimeMillis();
+
+        while (!driver.getTitle().toLowerCase().contains(title)
+                && System.currentTimeMillis() - startAt < 200) {
+            try {
+                Thread.sleep(5);
```

Comment:
Do NOT add 5 second sleeps. Any waiting for elements has to pull periodically and not just sleep for a long time (5 seconds * 1000 tests is a long time!).